plasmapy/particles/ionization_state_collection.py

```diff
@@ -630,7 +633,9 @@ def ionic_fractions(self, inputs: Union[Dict, List, Tuple]):
             try:
                 _particle_instances = [Particle(particle) for particle in inputs]
             except (InvalidParticleError, TypeError) as exc:
-                raise AtomicError("Invalid inputs to IonizationStates.") from exc
+                raise ParticleError(
```

Comment:
I'd rather hold off on adding these tests until the test helper refactoring in #728 is implemented, since one of the purposes of that refactoring is to simplify writing tests like these.  

 - [ ] After merging, create issue on improving test coverage in `ionization_state.py` and `ionization_state_collection.py`, and state that this should be done after the test helper refactoring is complete.